### ISSUE TITLE
Code gets lost when you press enter!

### ISSUE THREAD

**Issue by flbulgarelli** [uqbar-project/wollok#680]
I was typing the following: 

```wollok
/**
 * This is an example wollok hello world
 */
object ex {
	
	method sayHi(){
		
		console.println('Hello Wollok!')
	}
	
	method sayBaz() {
		
	}
}


class Foo {
	
}

class Bar inherits Foo {
	method zaraza() {
		
	}	
}
```

Then I positioned the cursor after the  `}` of `sayBaz`. I loose the method. 

```wollok
/**
 * This is an example wollok hello world
 */
object ex {
	
	method sayHi(){
		
		console.println('Hello Wollok!')
	}
	

}


class Foo {
	
}

class Bar inherits Foo {
	method zaraza() {
		
	}	
}
```

Then I selected the code. The method went back, but extra newlines where inserted between `sayHi` and `sayBaz`.

I was not always able to reproduce the issue. It seems that the code is actually there, but the editor is not rendering it propertly. 

I am using XUbuntu, and Wollok 1.4.3

**Comment by flbulgarelli** [comment]
I am adding the major severity tag, I had this problem or similar ones a lot of times during the day and makes it really hard to use the tool. 

I also reproduced it by copy pasting code or sometimes while formatting.

**Comment by fdodino** [comment]
It's related with xtext.
Could it be...
https://www.eclipse.org/forums/index.php/t/1070147/

It happens also in Wollok-Dev environment.

**Comment by flbulgarelli** [comment]
Does it happen to you frequently, @fdodino ? In ubuntu is happening all the time to me, it is driving me mad :disappointed:

**Comment by fdodino** [comment]
Using Linux Mint it doesn't happen all the time. I'd say rarely.

But it does happen also in my wollok-dev environment.

**Comment by JuanchiRios** [comment]
Same happen to me in fedora 23. I was refactoring some test and when i was writting it was like a get duplicate lines. But i thought it was kind of just a render stuff because if i kept writting the duplicate line disappeared

**Comment by javierfernandes** [comment]
To be merged soon in  #793

**Comment by javierfernandes** [comment]
Fixed in 1.5.0
